Assets/Scripts/ListComponents/TasksView/TasksController.cs

```diff
@@ -1,16 +1,21 @@
 using System;
+using System.Collections;
+using UnityEngine;
 
 
 
 // This class
 public class TasksController : ListController<Task>
 {
-    TasksUIController TasksUIController;
-
+    TasksUIController tasksUIController;
+    FocusController focusController;
 
     void Awake()
     {
+        focusController = GameObject.Find("Focus").GetComponent<FocusController>();
```

Comment:
Same advice here. Always better to just assign the reference via the unity inspector. If you want to make sure it's private but still able to see it in unity inspector, then you can add the attribute [SerializeField] before it.

Reply:
changed